lib/gui/pages/main/controllers/flash.js

```diff
@@ -60,69 +58,33 @@ module.exports = function(
     // otherwise Windows throws EPERM
     DriveScannerService.stop();
 
-    AnalyticsService.logEvent('Flash', {
-      image,
-      drive,
-      unmountOnSuccess: settings.get('unmountOnSuccess'),
-      validateWriteOnSuccess: settings.get('validateWriteOnSuccess')
-    });
-
     ImageWriterService.flash(image.path, drive).then(() => {
-      if (flashState.wasLastFlashCancelled()) {
-        AnalyticsService.logEvent('Elevation cancelled', {
-          image,
-          drive
-        });
-        return;
+      if (!flashState.wasLastFlashCancelled()) {
+        OSNotificationService.send('Success!', messages.info.flashComplete());
+        $state.go('success');
       }
-
-      OSNotificationService.send('Success!', messages.info.flashComplete());
-      AnalyticsService.logEvent('Done', {
-        image,
-        drive
-      });
-      $state.go('success');
     })
     .catch((error) => {
       OSNotificationService.send('Oops!', messages.error.flashFailure());
 
+      // TODO: All these error codes to messages translations
+      // should go away if the writer emitted user friendly
+      // messages on the first place.
```

Comment:
I see what you mean now. I think we can provide way better errors messages from `etcher-image-write` that don't include the word "Etcher", and thus would fit nicely in the CLI.